### PR TITLE
[INLONG-2051] audit query support mysql and elasticsearch

### LINKED ISSUE #2051: [Feature] [InLong audit] Add Audit API for Manager
### Description

This is a sub issue for releasing InLong audit of api.
image

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by yunqingmoswu**
### Title Name: [INLONG-2051] audit query support mysql and elasticsearch

fix #2051

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-web/src/main/resources/application-dev.properties [hunk 1]
```diff
@@ -45,24 +42,31 @@ spring.datasource.druid.testOnReturn=false
...
 # Application name in Sort
 sort.appName=inlong_app
-
 # Pulsar admin URL
 pulsar.adminUrl=http://127.0.0.1:8080,127.0.0.2:8080,127.0.0.3:8080
 # Pulsar broker address
 pulsar.serviceUrl=pulsar://127.0.0.1:6650,127.0.0.1:6650,127.0.0.1:6650
 # Default tenant of Pulsar
 pulsar.defaultTenant=public
+# Elasticsearch config
```
Comment (dockerzhang) [inline inlong-manager/manager-web/src/main/resources/application-dev.properties:62]: how to distinguish using MySQL or ES?

File: inlong-manager/manager-web/src/main/resources/application-prod.properties [hunk 2]
```diff
@@ -44,24 +41,31 @@ spring.datasource.druid.testOnReturn=false
...
 # Application name in Sort
 sort.appName=inlong_app
-
 # Pulsar admin URL
 pulsar.adminUrl=http://127.0.0.1:8080,127.0.0.2:8080,127.0.0.3:8080
 # Pulsar broker address
 pulsar.serviceUrl=pulsar://127.0.0.1:6650,127.0.0.1:6650,127.0.0.1:6650
 # Default tenant of Pulsar
 pulsar.defaultTenant=public
+# Elasticsearch config
```
Comment (dockerzhang) [inline inlong-manager/manager-web/src/main/resources/application-prod.properties:61]: ditto

Comment (yunqingmoswu) [reply inlong-manager/manager-web/src/main/resources/application-prod.properties:64]: It has done

Comment (yunqingmoswu) [reply inlong-manager/manager-web/src/main/resources/application-prod.properties:64]: It has done by add config item,The following are the specific configuration item: Audit query source that decide what data source to query, currently only supports [MYSQL|ELASTICSEARCH]

**Review by dockerzhang — APPROVED** [review]

**Review by doleyzi — APPROVED** [review]

**Review by aloyszhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Pull request merged**